packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -108,11 +112,50 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   /* @conditional-compile-remove(rooms) */
   mobileWithPreview = mobileWithPreview && rolePermissions.cameraButton;
 
+  /* @conditional-compile-remove(call-readiness) */
+  const [isDrawerShowing, setIsDrawerShowing] = useState(true);
+  /* @conditional-compile-remove(call-readiness) */
+  const onLightDismissTriggered = (): void => {
+    // do nothing here
+    // only way to dismiss this drawer is clicking on allow access which will leads to device permission prompt
+  };
+  /* @conditional-compile-remove(call-readiness) */
+  const drawerStyle: _DrawerSurfaceStyles = {
+    root: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      // apply zindex = 99 so drawer appear over device buttons and other components in the config page
+      zIndex: 99
+    }
+  };
+
   return (
     <Stack className={mobileView ? configurationContainerStyleMobile : configurationContainerStyleDesktop}>
       <Stack styles={bannerNotificationStyles}>
         <ErrorBar {...errorBarProps} />
       </Stack>
+
+      {
+        /* @conditional-compile-remove(call-readiness) */
+        mobileView && isDrawerShowing && (
+          <_DrawerSurface onLightDismiss={onLightDismissTriggered} styles={drawerStyle}>
+            <DomainPermissions
+              appName={'app'}
+              onTroubleshootingClick={() => console.log('clicked trouble shooting link')}
```

Comment:
remove console log here

Reply:
will link the ontroubleshootingclick with onpermissionTroubleShootingClick in another PR, need this one to be merged first https://github.com/Azure/communication-ui-library/pull/2433, in this pr I passed  onpermissionTroubleShootingClick down from composite to config screen

Reply:
This should be the troubleshootingClick prop from the composite itself I think?

Reply:
Just a nit, remove this line and then add it appropriately in #2433. Reason being another PR could be blocked or abandoned for any reason so we want to keep the codebase in a good state at all times (not always possible with stacked PRs though but something to strive for)

Reply:
fixed